great_expectations/__init__.py

```diff
@@ -1,3 +1,17 @@
+import sys
+
+PYTHON_SUPPORTED_VERSION = {"major": 3, "minor": {"lower": 8, "upper": 10}}
+if (sys.version_info.major != PYTHON_SUPPORTED_VERSION["major"]) or (
+    sys.version_info.major == PYTHON_SUPPORTED_VERSION["major"]
```

Comment:
Why do we care about major version at all?
Python 4 isn't going to be a thing, and users can't install us with Python 2?

Reply:
I guess I like to be explicit. People could install use via a `git clone` and not be gated via `pip`. I don't think python 2 users are going to use us but my preference is to leave the check in.